frontend/README.md

```diff
@@ -0,0 +1,51 @@
+# frontend-app
+
+This is a [sidebase merino](https://sidebase.io/) app created by running `npm create sidebase@latest`. This project uses the following technologies for a great developer- and user-experience:
+- [TypeScript](https://www.typescriptlang.org/)
+- [Nuxt 3](https://nuxt.com)
+- Tailwind CSS
+- Naive UI
+- GitHub Actions based CI
+
+## How to get going?
+
+This is a straight-forward setup with minimal templating and scaffolding. The options you selected during the sidebase CLI setup are all here though. Good places to continue reading are:
+- [the First Steps documentation](https://sidebase.io/sidebase/usage)
+- [our discord](https://discord.gg/auc8eCeGzx)
```

Comment:
I'm not sure if this was copied over, but seems like term "our discord" in an opensource project for maker infrastructure might cause confusion

Reply:
Thanks for spotting it, I rewrote the whole readme now